lua/nvim-lsp-installer/server.lua

```diff
@@ -66,6 +67,28 @@ function M.Server:get_default_options()
     return vim.deepcopy(self._default_options)
 end
 
+function M.Server:get_supported_filetypes()
+    log.debug("got filetypes query request for: " .. self.name)
+
+    if self._default_options.filetypes then
+        return self._default_options.filetypes
+    end
+
+    local configs = require "lspconfig/configs"
+    local default_server_config = "lspconfig/" .. self.name
+    pcall(require, default_server_config)
```

Comment:
I believe by interacting with lspconfig like this, it will cause the server to show up in `:LspInfo` (which has caused some confusion before, especially for servers that aren't installed). Since this would be called for every server in the UI, it'd cause all LSP servers to show up in `:LspInfo`

Reply:
I'm suspecting this might be the case, which is why I reset `lspconfig/configs` and why I added the note about the possibility of doing this lookup once and caching the values in a file you that renew every once in a while. That might also allow you to remove this list: https://github.com/williamboman/nvim-lsp-installer/blob/ada75fbbad96f63fe5cd7ba3e44d5235ddf33ca6/lua/nvim-lsp-installer/servers/init.lua#L41

Having said that, I don't have any problems with triggering `:LspInfo` after `:LspInstallInfo`, but I've noticed some weird error, about an invalid name, that shows up once or twice, so your YMMV 😄